client/src/components/GameEnv/GameEnv.jsx

```diff
@@ -78,7 +95,24 @@ const GameEnv = ({ setNav }) => {
                 yourSlots[i].point_health -= (val.point_attack - yourSlots[i].point_armor);
               }
             } else if (val.point_attack) {
-              hp -= val.point_attack;
+              let counter = true;
+              for (let j = 0; j <= 3; j += 1) {
+                if (yourSlots[j]) {
```

Comment:
it appears to me, that all of the following if statements need to be true, or the function will end, if so, you could probably condense this all to one if statement using the && operator

Reply:
i tried to changed it but the (.incudes()) throw an error if yourslots is undefined